src/tools/pub/pubspec.md

```diff
@@ -272,6 +277,39 @@ publish_to: none
 {% endprettify %}
 
 
+### False_secrets
+
+When publishing leak detection on the client will abort publishing if it finds
+potential leaks of secret credentials, API keys and cryptographic keys in files
+to be published.
+While detection of possible secrets relies on advanced heuristics using regular
```

Comment:
Maybe 'advanced' sounds a bit too much for basically running though a bunch of regexps and doing a bit of checking on matches.

Reply:
We do have entropy requirements, but just saying heuristics is fine.